spec/services/fees/subscription_service_spec.rb

```diff
@@ -318,15 +317,15 @@
 
         it 'creates a fee' do
           result = fees_subscription_service.create
-          created_fee = result.fee
-
-          aggregate_failures do
-            expect(created_fee.id).not_to be_nil
-            expect(created_fee.invoice_id).to eq(invoice.id)
-            expect(created_fee.amount_cents).to eq(plan.amount_cents)
-            expect(created_fee.amount_currency).to eq(plan.amount_currency)
-            expect(created_fee.units).to eq(1)
-          end
+
+          expect(result.fee).to have_attributes(
+            id: String,
+            invoice_id: invoice.id,
+            amount_cents: plan.amount_cents,
+            amount_currency: plan.amount_currency,
+            unit_amount_cents: plan.amount_cents,
+            units: 1
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)